tests/configurable-timeout-test.ts

```diff
@@ -36,11 +36,11 @@ QUnit.test('We can use a custom later', function(assert) {
     }
   });
 
-  bb.scheduleOnce('one', () => {
+  bb.later(() => {
```

Comment:
had to change to `later` because I remove `setTimeout` fallback from `next` so the test was failing